R/model_dashboard.R

```diff
@@ -45,7 +45,7 @@
 #' particular the `check`-argument can help, to skip some unnecessary checks).
 #'
 #' @examples
-#' if (FALSE) {
+#' if (interactive()) {
```

Comment:
> Functions which are supposed to only run interactively (e.g. shiny)
should be wrapped in if(interactive()). This does not seem necessary for
your functions

Reply:
Apparently `if(FALSE)` is also refused by CRAN: https://github.com/ThinkR-open/prepare-for-cran#about-dontrun. I think we can use `\dontrun` because the examples take more than 5 seconds and require external software (pandoc)